Taskfile.yml

```diff
@@ -0,0 +1,87 @@
+version: "3"
+
+includes:
+  utils: "tools/yscope-dev-utils/taskfiles/utils.yml"
+
+vars:
+  # Paths
+  G_BUILD_DIR: "{{.ROOT_DIR}}/build"
+  G_EMSCIPRTEN_DIR: "{{.G_BUILD_DIR}}/emsdk"
```

Comment:
How about `G_EMSDK_DIR` and `emsdk` for the task? (If not, let's fix the spelling.)